apps/Unciv/install

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+### Make directory ~/Unciv if not available
+if [[ ! -d ~/Unciv ]]; then
+  mkdir ~/Unciv || error 'Failed to make folder $HOME/Unciv!'
+fi
+
+### Add LICENSE from Github if doesn't exist
+if [[ ! -f ~/Unciv/LICENSE ]]; then
+  wget -O ~/Unciv/LICENSE https://raw.githubusercontent.com/yairm210/Unciv/master/LICENSE || error 'Failed to make folder $HOME/Unciv!'
+fi
+
+### install openjdk-8-jre-headless
+# the 'default jre' (OpenJDK-11-jre) package takes too much RAM and runs sloppy in Raspberry Pi 3B+
+# so, we will use a minimal OpenJDK 8 JRE installation to run this.
+"${DIRECTORY}/pkg-install" openjdk-8-jre-headless || exit 1
```

Comment:
Rejected cause `default-jre` comes with **Raspbian Desktop with Recommended Software**, if I remeber correctly. If we run **Unciv** with it the games becomes very unplayable. Since I have tried it myself I know how bad it runs. So, we are going to install `openjdk-8-jre-headless` and run **Unciv**. The performance improvement can;t be ignored here. There is difference between a playable game and an unplayable game and the difference can't be ignored.
**Unciv** relies on **Java 8** so a minimal **JRE** is required to be downloaded to ensure that it runs.